### PR TITLE
[FIX] flake8

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+[report]
+omit =
+    /usr/*
+    */bin/*
+    */lib/*
+    */odoo/*
+    */openerp/*
+    */tests/*
+    *__init__.py
```
Comment (bwrsandman) [inline .coveragerc:8]: A better regex for filenames is:

```
[report]
include =
    */OCA/web/*

omit =
    */tests/*
    *__init__.py
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - git clone https://github.com/gurneyalex/maintainer-quality-tools.git $HOME/maintainer-quality-tools
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - $HOME/maintainer-quality-tools/travis/travis_install_nightly 6.1
```
Comment (bwrsandman) [inline .travis.yml]: can simply be

```
- travis_install_nightly 6.1
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ - pip install coveralls flake8
+
+services:
+  - postgresql
```
Comment (bwrsandman) [inline .travis.yml]: services section can be taken out, it is not necessary

File: web_ckeditor4/__openerp__.py [hunk 4]
```diff
@@ -51,60 +51,60 @@
...
+        #'static/lib/trunk/core/creators/themedui.js',
+        #'static/lib/trunk/core/editable.js',
+        #'static/lib/trunk/core/selection.js',
+        #'static/lib/trunk/core/style.js',
+        #'static/lib/trunk/core/dom/comment.js',
+        #'static/lib/trunk/core/dom/elementpath.js',
+        #'static/lib/trunk/core/dom/text.js',
+        #'static/lib/trunk/core/dom/rangelist.js',
+        #'static/lib/trunk/core/skin.js',
+        #'static/lib/trunk/core/_bootstrap.js',
```
Comment (bwrsandman) [inline web_ckeditor4/__openerp__.py:107]: Out of curiosity, what is this?

Comment (hbrunn) [inline web_ckeditor4/__openerp__.py:107]: If you ever have to debug into ckeditor code, the minified monolithic js-file won't help you much, so here is a listing of all the files it is built from. So pure developer convenience.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+[![Build Status](https://travis-ci.org/OCA/web.svg?branch=6.1)](https://travis-ci.org/OCA/web)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/web.svg)](https://coveralls.io/r/OCA/web?branch=6.1)
```
Comment (bwrsandman) [inline README.md]: Using the png bage for coverage allows to specify which branch to show.

```
[![Coverage Status](https://coveralls.io/repos/OCA/web/badge.png?branch=6.1)](https://coveralls.io/r/OCA/web?branch=6.1)
```

[Coverage Status](https://coveralls.io/r/OCA/web?branch=6.1)

File: web_ckeditor4/__openerp__.py (hunk 4 above)
Comment (bwrsandman) [inline web_ckeditor4/__openerp__.py:107]: Does odoo minify js files on its own now?

Comment (hbrunn) [inline web_ckeditor4/__openerp__.py:107]: The issue is more the way ckeditor works: You compile one ckeditor.js file from a bunch of modules as you see fit via a build script, and back then I couldn't find a switch that would suppress the minifying step.

**Comment by bwrsandman** [comment]
:+1: 

LGTM

**Comment by hbrunn** [comment]
Concerning your other tips (thanks for them): https://github.com/OCA/maintainers-tools/wiki/Migration-Launchpad-%E2%86%92-GitHub proposes to pick the relevant files from connector resp. maintainer-quality tools, so I propose to either point the wiki to better versions of the files or change it in those branches, otherwise you'll have to write the same review over and over again

**Comment by bwrsandman** [comment]
I have a PR to fix this, unfortunately, I don't have the rights to merge.
https://github.com/OCA/maintainer-quality-tools/pull/19

**Comment by hbrunn** [comment]
me neither, but I gave you a thumbs up at least

**Comment by guewen** [comment]
On 07/07/2014 03:43 PM, Holger Brunn wrote:

> me neither, but I gave you a thumbs up at least

Only members of https://github.com/orgs/OCA/teams/owners have commit
rights I guess.
That's a bit sad because there are several pending PR that could help us
all to integrate these tools.

**Comment by yvaucher** [comment]
A board member needs to set correct rights on https://github.com/OCA/maintainer-quality-tools/

**Pull request merged**